dlfcn.c

```diff
@@ -208,6 +208,157 @@ static BOOL MyEnumProcessModules( HANDLE hProcess, HMODULE *lphModule, DWORD cb,
     return EnumProcessModulesPtr( hProcess, lphModule, cb, lpcbNeeded );
 }
 
+// Create a converted multibyte character string from a wide character string.
+static char *utf8_encode( const wchar_t *wstr )
+{
+    size_t input_str_len = wcslen( wstr );
+    if ( !wstr || input_str_len == 0 )
+    {
+        return NULL;
+    }
+    int size_needed = WideCharToMultiByte( CP_UTF8, 0, wstr, input_str_len, NULL, 0, NULL, NULL );
+    char *to_return = (char *) malloc( size_needed * sizeof( char ) );

```

Comment:
`sizeof(char)` is by definition 1. Explicit usage of `sizeof(char)` is suspicious.

Reply:
Even though it is 1 by definition, it is better to be explicit, since I am returning a `char*` array.